modules/installation-about-custom-aws-vpcs.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * installing/
+
+[id="installation-about-custom-aws-vpcs_{context}"]
+= About using a custom VPC
+
+In {product-title} {product-version}, you can deploy a cluster into an existing VPC in Amazon Web Services (AWS). If you do, you must also use existing subnets within the VPC and routing rules.
+
+By deploying {product-title} into an existing AWS VPC, you might be able to avoid limit constraints in new accounts or more easily abide by the operational constraints that your company's guidelines set. This is a good option to use if you cannot obtain the infrastructure creation permissions that are required to create the VPC yourself.
+
+[id="installation-about-custom-aws-vpcs-requirements_{context}"]
+== Requirements for using your VPC
+
+The installation program will no longer create the following components:
+* Internet gateways (`aws_internet_gateway`)
+* NAT gateways (`aws_nat_gateway`, `aws_eip.nat_eip`)
+* Subnets (`aws_subnet`)
+* Route tables (`aws_route_table`, `aws_route`, `aws_route_table_association`, `aws_main_route_table_association`)
+* VPCs (`aws_vpc`)
+* VPC DHCP options (`aws_vpc_dhcp_options`, `vpc_dhcp_options_association`)
+* VPC endpoints (`aws_vpc_endpoint`)
+
+If you use a custom VPC, you must correctly configure it and its subnets for the installation program and the cluster to use. The installation program cannot subdivide network ranges for the cluster to use, set route tables for the subnets, or set VPC options like DHCP, so you must do so before you install the cluster.
+
+Your VPC must meet the following characteristics:
+
+* The VPC’s CIDR block must contain the `Networking.MachineCIDR` range, which is the IP address pool for cluster machines.
+* The VPC must not use the `kubernetes.io/cluster/.*: owned` tag. The installation program modifies your subnets to add the `kubernetes.io/cluster/.*: shared` tag. Your subnets must have at least one free tag slot available for it. Review the current link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/Using_Tags.html#tag-restrictions[Tag Restrictions] in the AWS documentation to ensure that the installation program can add a tag to each subnet that you specify.
+* You must enable the `enableDnsSupport` and `enableDnsHostnames` attributes in your VPC so that the cluster can use the Route53 zones that are attached to the VPC to resolve cluster’s internal DNS records. See link:https://docs.aws.amazon.com/vpc/latest/userguide/vpc-dns.html#vpc-dns-support[DNS Support in Your VPC] in the AWS documentation.
+
+
+If you us a cluster with public access, you must create a public and a private subnet for each availability zone that your cluster uses. No other service can use these subnets.
```

Comment:
> If you us a cluster with public access

`us` -> `need` I think

> No other service can use these subnets.

the goal is to share, so this proabably won't fly and we don't need this.

Reply:
s/us/use

I thought that the subnets needed to be reserved for OCP itself. Maybe s/service/service external to the cluster?